cmd/snap-exec/main_test.go

```diff
@@ -98,15 +103,31 @@ func (s *snapExecSuite) TestFindCommand(c *C) {
 	info, err := snap.InfoFromSnapYaml(mockYaml)
 	c.Assert(err, IsNil)
 
+	snapPath := fmt.Sprintf("%s/snapname/unset", dirs.SnapMountDir)
+
 	for _, t := range []struct {
-		cmd      string
-		expected string
+		app          string
+		cmd          string
+		includeChain bool
+		expected     string
 	}{
-		{cmd: "", expected: `run-app cmd-arg1 $SNAP_DATA`},
-		{cmd: "stop", expected: "stop-app"},
-		{cmd: "post-stop", expected: "post-stop-app"},
+		// With command chain
+		{app: "app", cmd: "", includeChain: false, expected: fmt.Sprintf("%s/run-app cmd-arg1 $SNAP_DATA", snapPath)},
+		{app: "app2", cmd: "", includeChain: false, expected: fmt.Sprintf("%[1]s/chain1 %[1]s/chain2 %[1]s/run-app2", snapPath)},
```

Comment:
Heh, I always disliked this syntax, although not sure if there's a nicer way to do this. Feel free to ignore me being grumpy about this.

Reply:
Haha, no strong feelings here, I just added the duplication to get rid of the ugly syntax.